### ISSUE TITLE
a11y: invalid nested headings

### ISSUE THREAD

**Issue by eatyourgreens** [OxfordRSE/gutenberg#355]
The home page headings are a mixture of h5 and h3 (tested on an iPhone with WAVE.)

https://wave.webaim.org/report#/https://train.rse.ox.ac.uk/

Heading structure should be h1 > h2 > h3, with only one h1 at the top of the heading hierarchy.

**Comment by eatyourgreens** [comment]
In a screen reader, ‘Course Materials’ is a subheading under ‘Workflows with Snakemake’, and very easy to miss.

**Comment by eatyourgreens** [comment]
For the home page, maybe the following structure:

- h1: OxRSE Training
  - h2: Events
    - h3: Event 1
    - h3: Event 2
  - h2: Course Material
